apps/jobs/tasks.py

```diff
@@ -145,6 +145,9 @@ def async_job_fire(self):
             task = None
             try:
                 task = AsyncTask(async_job_id)
+                while cache.ttl("REPLICATE_SHIPMENTS_LOCK") is None:
```

Comment:
This seems overly complicated for what we need.  In this situation I expect us to check the existence of the key, not refer to its TTL.  The key should be set initially with no TTL ,so it will persist as long as we need it to, then when the management command is complete, that key can be deleted.